src/components/PriceBar/TileColorBoard.tsx

```diff
@@ -7,7 +7,8 @@ import priceFormat from "@/utils/priceFormat";
 
 const TileColorBoard: React.FC = () => {
   const tileBlocks = useStoreSelector((state) => state.priceBar.blocks);
-  const court = useStoreSelector((state) => state.courtSize);
+  // const court = useStoreSelector((state) => state.courtSize);
```

Comment:
It seems replaced by the next line. Please remove it if not using or add a TODO.